src/utils/file_ops.cpp

```diff
@@ -144,14 +144,14 @@ void mp::FileOps::open(std::fstream& stream, const char* filename, std::ios_base
     stream.open(filename, mode);
 }
 
-std::unique_ptr<std::ostream> mp::FileOps::open_write(const fs::path& path) const
+std::unique_ptr<std::ostream> mp::FileOps::open_write(const fs::path& path, std::ios_base::openmode mode) const
```

Comment:
Shouldn't this have a default open mode?

Reply:
There are default open modes defined in the header file :smile: